src/java/org/apache/cassandra/metrics/RepairMetrics.java

```diff
@@ -18,17 +18,75 @@
 
 package org.apache.cassandra.metrics;
 
+import java.util.Collections;
```

Comment:
This is from https://issues.apache.org/jira/browse/CASSANDRA-18952 and no changes are from this PR... can ignore this file